src/ChargeClustering/ChargeClustering.jl

```diff
@@ -77,19 +76,12 @@ function cluster_detector_hits(
 end
 
 
-function cluster_detector_hits(
-    table::TypedTables.Table, 
-    cluster_radius::PT, 
-    cluster_time::TTT
-) where {PT <: RealQuantity, TTT <: RealQuantity}
-    @assert :pos in TypedTables.columnnames(table) "Table has no column `pos`"
-    @assert :thit in TypedTables.columnnames(table) "Table has no column `thit`"
-    @assert :edep in TypedTables.columnnames(table) "Table has no column `edep`"
-    @assert :detno in TypedTables.columnnames(table) "Table has no column `detno`"
+function cluster_detector_hits(table::TypedTables.Table, cluster_radius::RealQuantity, cluster_time::RealQuantity = Inf * u"s")
+    @assert is_detector_hits_table(table) "Table does not have the correct format"
     
     # Collect all time-clustered results
     all_results = []

```

Comment:
We should avoid declaring `Arrays` with unknown element-type, as this will default to `Any` and not be type-stable.